src/main/java/com/byznass/tiolktrack/postgres/dao/PostgresGpsPersister.java

```diff
@@ -0,0 +1,50 @@
+package com.byznass.tiolktrack.postgres.dao;
+
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
+import com.byznass.tiolktrack.kernel.dao.GpsPersister;
+import com.byznass.tiolktrack.kernel.model.gps.Gps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class PostgresGpsPersister implements GpsPersister {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(PostgresGpsPersister.class);
+
+	private final Connection connection;
+
+	@Inject
+	public PostgresGpsPersister(Connection connection) {
+
+		this.connection = connection;
+	}
+
+	@Override
+	public void persist(Gps gps) throws TiolkTrackException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)